src/cursor.js

```diff
@@ -237,8 +237,12 @@ export default class Cursor {
 
   // Create an element with the correct ownerWindow
   // (see: http://www.w3.org/DOM/faq.html#ownerdoc)
-  createElement (tagName) {
-    return this.win.document.createElement(tagName)
+  createElement (tagName, attributes = []) {
+    const element = this.win.document.createElement(tagName)
+    for (const {attributeName, attributeValue} of attributes) {
```

Comment:
I think that an object for the attributes is more common.
It's closer to the dom api. I'd prefer that. `attributes: {class: 'foo'}`